changelog.d/3533.bugfix

```diff
@@ -0,0 +1 @@
+Fix queueing of federation requests
```

Comment:
I feel like this is too vague to go into a changelog. Something like "Fix queued federation requests being processed in the wrong order", might be better maybe?